dietpi/dietpi-software

```diff
@@ -13434,6 +13461,15 @@ _EOF_
 
 		fi
 
+#--------------------------------
+		software_id=142 # microk8s
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+
+			Banner_Uninstalling
+
+			G_EXEC snap uninstall microk8s 
```

Comment:
```suggestion
			command -v snap > /dev/null && G_EXEC_OUTPUT=1 G_EXEC snap remove microk8s 
```
It's "remove" not "uninstall", and we shouldn't error out when Snap itself is not installed anymore.

Reply:
Thought I had fixed it in the IDE but it seems I only fixed it on dietpi-software file on the pi4 and not the IDE